output.proto

```diff
@@ -64,8 +64,13 @@ message SystemHealth {
       SENSOR_TILTED = 3;
       SENSOR_SUSPENDED = 4;
     }
+    message SensorInfo {
+      string topic = 1;
+      string sensor_name = 2;
+      SensorStatus status = 3;
+    }
     Status status = 1;
-    map<string, SensorStatus> sensors = 3;
```

Comment:
@thor-sr @sebastian-seoul-robotics FYI, we output pairs of ROS topic and sensor status, where users have no idea about which topic corresponds to which lidar. Alex is changing the key to use the UID and putting extra info (e.g., lidar name in SENSR and topic name for backward compatibility). Here's the link to the relevant conversation: https://seoulrobotics.atlassian.net/browse/BMW-631?focusedCommentId=32983